package.json

```diff
@@ -23,10 +23,15 @@
     "@nestjs/common": "^10.0.0",
     "@nestjs/core": "^10.0.0",
```

Comment:
**issue:** please note that you pushed the `node_modules` folder alongside with your other project files. You should remove it and add it to a `.gitignore` file so you don't push it by mistake in the future.


**Reasoning:** In the `node_modules` folder exist the project dependencies (e.g.: libraries, etc), each developer should install them (npm i) independently on their own machine (due to the difference of that machine (e.g.: OS)) so you don't have conflicts in the future. In order to ensure that each developer has a specific version of a library there exists the `package-lock.json` file.

Reply:
Oh.. Yes, I actually know about this but didn't notice it this time. Thank you for pointing it out!

![image](https://github.com/user-attachments/assets/22843abd-ce4e-446d-947a-c4e813658156)

It seems I already had "/node_modules" included in the .gitignore, but I also added these lines now:
`#Node modules
node_modules
node_modules/`

and removed node_modules using this command `git rm -r --cached node_modules/`.
I think it should be fine now.